Cabal/Distribution/Compat/Internal/TempFile.hs

```diff
@@ -40,6 +44,12 @@ import qualified System.Posix
 -- needs to be changed to make this better.
```

Comment:
Do you mind updating the comment above, I guess `base` library *is changed*?

Reply:
yeah, will do